lib/beaker-hostgenerator/data.rb

```diff
@@ -147,6 +147,15 @@ def osinfo
             'template' => 'amazon-7-arm64'
           }
         },
+        'archlinux2-64' => {
```

Comment:
the `2` was there because facter assumed, based on kernel `2.x` that the major version of archlinux is 2. the current kernel version is 5. I don't know if it matters which number is in the data.rb, but the correct current one would be 5.

Reply:
Ah, good to know what it's based on. I just copied it from voxpupuli's repo. I'm wondering if we can just leave off the number altogether.